src/Account/CreditClient.php

```diff
@@ -36,6 +36,20 @@ public function getAll()
         }, $body->data);
     }
 
+    /**
+     * Get credits for an individual service
+     * @param $reference
+     * @return Credit
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function findByReference($reference)
```

Comment:
Should this be `getByReference()` to match our naming conventions?

Reply:
Good point, I will make the changes now.

Reply:
@phily245 this has now been updated